### ISSUE TITLE
Feature request: TOR support

### ISSUE THREAD

**Issue by Stadicus** [neogeno/shango-lightning-wallet#37]
When setting up a on-premise Lightning full node that is accessable from the outside, one major hurdle is to create certificates, preferably using a dyndns domain name & config the router port forwarding correctly. 

One of the major benefits of running Lightning over Tor is that you get your onion address and that takes care for everything. The additional benefit of not revealing its own IP address is very nice as well, of course.

Are there plans to support TOR connectivity?

**Comment by neogeno** [comment]
Yes TOR support is a important feature that will be implemented after the app goes Public beta, expected in 2 weeks. Thanks for the feedback!

**Comment by gabrieldov** [comment]
Hi guys, I believe I broke the Raspiblitz TOR by following the instructions here to create a compatible certificate... I removed the tls.cert and followed the instructions. Now my node cannot be used with the newly created certificate. How do I create a compatible tls.cert to be able to interact with my node again?

**Comment by gabrieldov** [comment]
Hi guys, I'm trying to run Shango with the TOR Raspiblitz without success... Raspiblitz still provides me with the QR code that has my local IP - and Shango is unable to connect to it. Before I activated TOR within Raspiblitz, Shango was working perfectly. Now, nada.

**Comment by thibautrey** [comment]
Any news on that topic? I'm running raspiblitz behind a 4g network. Therefore public IP isn't an option. TOR, would be great for that matter.

**Comment by bbelo** [comment]
I'm also waiting patiently for this :-)
